ocm-container.sh

```diff
@@ -17,7 +17,16 @@ EOF
 }
 
 ARGS=()
-BUILD_TAG="latest"
+if [[ -n "${OCM_CONTAINER_TAG}" ]]; then
+  BUILD_TAG="${OCM_CONTAINER_TAG}"
+else
+  BUILD_TAG="latest"
+fi
+if [[ -n "${OCM_CONTAINER_REPO}" ]]; then
+  BUILD_REPO="${OCM_CONTAINER_REPO}"
+else
+  BUILD_REPO="localhost"
+fi
```

Comment:
I very-much dislike `else` blocks unless they're absolutely needed.  Can we potentially either use bash parameter substitution logic here like 

```
BUILD_TAG="${OCM_CONTAINER_TAG:=latest}"
```

 or if we're worried about that functionality we could do something like

```
BUILD_TAG="latest"
BUILD_REPO="localhost"

if [[ -n "${OCM_CONTAINER_TAG}" ]]; then
  BUILD_TAG="${OCM_CONTAINER_TAG}"
fi
... etc
```

Reply:
Aha! Thanks Kirk!
I was trying to work out how to do it that way!
I will get that fixed. :)